data/_ui/help.txt

```diff
@@ -161,6 +161,10 @@ help "uninstalling and storage"
 	`You can also uninstall outfits directly into planetary storage by selecting the outfit and pressing U. This can be useful for if you want to shuffle outfits between ships, store outfits for future use, or simply try on different outfits. Outfits in storage can also be transferred to cargo when you go to 'purchase to cargo' mode. Outfits that are in planetary storage or cargo can be (re)installed at any outfitter without cost.`
 	`If you sell an outfit at an outfitter, then you can immediately buy it back for the same price that you sold it, so long as you do not leave the planet. This means that you do not necessarily need to worry about uninstalling outfits into planetary storage when you are changing outfits around.`
 
+help "outfitter with multiple ships"
+	`In the outfitter you can manage multiple ships at the same time. Ctrl+click a ship icon to add it to your selection. You can also use Shift+click to select a range of ships.`
```

Comment:
```suggestion
	`You can manage outfits on multiple ships at the same time. Ctrl+click a ship icon to add it to your selection. You can also use Shift+click to select a range of ships.`
```
It should be fairly clear by now that this is the outfitter and "manage" doesn't alone necessarily imply "outfits" in my opinion. When I think of "managing" ships, I think of the player info panel and adjusting the order or parking status of ships.